src/Fantomas/TriviaContext.fs

```diff
@@ -7,7 +7,7 @@ open Fantomas.Context
 open Fantomas.TriviaTypes
 
 let tok (range: range) (s: string) =
-    enterNodeToken range +> (!-s) +> leaveNodeToken range
+    enterNode range +> (!-s) +> leaveNode range
```

Comment:
I think better would be to make new function with this body, so we keep meaning of `tok` function as something dealing with Tokens.

Reply:
`tok` was only used in one place so that is why I just changed the implementation.
Made the new function you asked.